packages/react-native-bpk-theming/src/util-test.js

```diff
@@ -115,18 +123,14 @@ describe('makeThemePropType', () => {
       ).toBeFalsy();
     });
 
-    it('should fail when random props are provided', () => {
+    it('should not fail when random props are provided', () => {
```

Comment:
So, this test failed when I made my change. From what I can tell though, we really don't mind if random props are passed in, so I changed it.

Reply:
👍